### ISSUE TITLE
Internal server error: must be an instance of WP, instance of WPGraphQL\\Data\\NodeResolver given

### ISSUE THREAD

**Issue by suckydev** [wp-graphql/wp-graphql#1904]
Hi,

I'm doing a simple query:

```
query MyQuery {
  post(id: "example-post-3", idType: SLUG) {
    id
  }
}
```

<img width="1099" alt="Internal server error" src="https://user-images.githubusercontent.com/75957128/117492236-9c194880-af71-11eb-9a8d-ff6c0478241e.png">

and received the following error:

```
"debugMessage": "Argument 1 passed to AmpProject\\AmpWP\\PairedRouting::restore_path_endpoint_in_environment() must be an instance of WP, instance of WPGraphQL\\Data\\NodeResolver given, called in /wordpress/core/5.7.1/wp-includes/class-wp-hook.php on line 292",
"message": "Internal server error",
"extensions": {
    "category": "internal"
},
```

but everything worked fine just a few days ago--I haven't changed anything. I have no idea what this means. I use WordPress for Business on WordPress.com and installed the plugin there.

Any help would be really appreciated.

**Comment by adithwip** [comment]
I have the same issue :'(

This broke my Next.js build. Because I can't get the data from a single post based on slug as ID.

<img width="1440" alt="Screen Shot 2021-05-08 at 15 00 43" src="https://user-images.githubusercontent.com/27177332/117531724-2c0cd000-b00e-11eb-8707-b4ce66403627.png">

**Comment by suckydev** [comment]
@adithwip no, have you? I don't wanna create some weird workaround, no idea why this issue just appeared now, since I haven't changed anything.

**Comment by adithwip** [comment]
No, I have not. @suckydev 

I tried to notify in the Slack channel. We can wait for any contributor to noticed. Since I don't have PHP skills at the moment to fix it 😆

**Comment by jasonbahl** [comment]
👋🏻 this issue re-surfaced in #1979 and I was able to track it down and add a fix in #1992! 

Should be released soon for those that need to use WPGraphQL and AMPWP plugins together 😄
